com.unity.netcode.gameobjects/Prototyping/AssemblyInfo.cs

```diff
@@ -1,6 +1,6 @@
 using System.Runtime.CompilerServices;
 
-#if UNITY_EDITOR
+#if UNITY_EDITOR || UNITY_INCLUDE_TESTS
```

Comment:
I'm not sure you actually need/want this change. I was able to run the RuntimeTests standalone without this... it won't hurt anything to have it, but also seems not necessary. Same with the equivalent in `com.unity.netcode.gameobjects/Runtime/AssemblyInfo.cs`

Reply:
Gotcha, reverting.